app/konnect/dev-portal/customization/index.md

```diff
@@ -20,6 +26,16 @@ From the **Appearance** menu you have the ability to modify the following option
 
 For details on the requirements for each customizable option, hover over the information (`i`) icon next to each item in the UI.
 
+### Complete customization using the open source Dev Portal
```

Comment:
@rosskukulinski I added this section that contains info about how to configure the open source Dev Portal and some of the benefits. Was this what you had in mind?

Reply:
Great! Few small tweaks, but looking good.